packages/components/src/color-palette/index.native.js

```diff
@@ -237,17 +239,27 @@ function ColorPalette( {
 			} ) }
 			{ shouldShowCustomIndicator && (
 				<View
-					style={ styles.customIndicatorWrapper }
+					style={ [
+						styles.customIndicatorWrapper,
+						customIndicatorWrapperStyles,
```

Comment:
Can we move it to the const since the same styles are used also in other `View`?

Reply:
Thank you for reviewing this @lukewalczak :)
I agree with all your suggestions and submitted the changes